assets/questions/rle.json

```diff
@@ -92,7 +94,7 @@
         "acquired_skills": ["String Manipulation", "Sets", "Arrays", "Maps"],
         "input_function": null,
         "output_function": null,
-        "main_function": "sample.DecodeEncodedString",
+        "main_function": "DecodeEncodedString",
```

Comment:
Oh, sorry, now I'm confused. I thought this was the name of the main function in the student's code that is to be executed? Similar to abbreviate() in i18n.

Also, by my previous comment, I meant removing 'sample.' in i18n.json.